openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -212,7 +218,7 @@ def process(self, instance):
         self.log.info("Extract sets (%s) ..." % _scene_type)
         lookdata = instance.data["lookData"]
         relationships = lookdata["relationships"]
-        sets = relationships.keys()
+        sets = list(relationships.keys())
```

Comment:
Avoids the keys being a `dict_keys` object which the `lib.empty_sets` function does not allow - it asserts for list or tuple.